project_task_analytic_partner/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Partner in task works
+=====================
+
+This module adds the partner on task works and propagate it to the
+auto-generated analytic entries on the "Other partner" field.
+
+
```

Comment:
AFAICS this module requires HR Timesheet to work.
This is not clear from the title, and should be explained here.